main/templates/main/profile.html

```diff
@@ -1,13 +1,33 @@
-{% extends 'main/base.html' %}
+<html>
 {% load staticfiles %}
 
-{% block title %}
-Profile
-{% endblock %}
+<body>
 
+	<head>
+		<meta charset="utf-8" name="viewport" content=" width = device-width, initial-scale = 1.0">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<title>{% block title %}{% endblock %}- SMS 2019</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1">
 
-{% block body %}
-<body>
+		<link rel="stylesheet" type="text/css" href="{% static 'css/template.css' %}">
+		<link rel="stylesheet" type="text/css" href="{% static 'css/game.css' %}">		
+		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+		<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+		
+		<meta charset="utf-8" name="viewport" content=" width = device-width, initial-scale = 1.0">
+		<link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
+		<link href="https://fonts.googleapis.com/css?family=Exo" rel="stylesheet">
+		<link href="https://fonts.googleapis.com/css?family=Baloo+Chettan" rel="stylesheet">
+		<link href="https://fonts.googleapis.com/css?family=Sniglet" rel="stylesheet">
+		<link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+		<link href="https://fonts.googleapis.com/css?family=Muli" rel="stylesheet">
+		<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+		<link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+		<link href="https://fonts.googleapis.com/css?family=Roboto:900" rel="stylesheet">
+		<link href="https://fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet">
+		<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+	</head>
 
```

Comment:
Are these imports different from those in base.html?
Why don't the templates inherit from the base template?

Reply:
If we extend all templates from base.html, we would have to import all the js files in the base template. Now since every template extends from it, to load any template, every js file would need to run first. This would make the website slow. We do not need to run all the pages' js in every single page.

Reply:
You can add blocks in the head tag. The files that need to be imported for every page can be kept in base.html and for page-specific scripts, you can import them in the block.

Reply:
For now, I am merging this. We can work this out in a separate issue.